e2e/api/api_test.go

```diff
@@ -129,7 +129,7 @@ func (s *APITestSuite) TestCellsRemovedAfterSwitchAccount() {
 		}
 	)
 
-	config, err := e2e.MakeTestNodeConfig(params.StatusChainNetworkID)
+	config, err := e2e.MakeTestNodeConfig(e2e.GetNetworkID())
```

Comment:
We already have a `testing` package which we dot-import. Shall we move this function there?

Reply:
We can move it if you so wish.